sdk/trace/trace_test.go

```diff
@@ -552,11 +509,8 @@ func TestLinks(t *testing.T) {
 		ParentSpanID:    sid,
 		Name:            "Links/span0",
 		HasRemoteParent: true,
-		Links: []apitrace.Link{
-			{SpanContext: sc1, Attributes: []core.KeyValue{k1v1}},
-			{SpanContext: sc2, Attributes: []core.KeyValue{k2v2, k3v3}},
-		},
-		SpanKind: apitrace.SpanKindInternal,
+		Links:           nil,
```

Comment:
you need to add the links with StartSpan as below
```go
	span := startSpan(tp, "Links",
		apitrace.LinkedTo(sc1, key.New("key1").String("value1")),
		apitrace.LinkedTo(sc1,
			key.New("key2").String("value2"),
			key.New("key3").String("value3"),
		),
	)
```
and then check for non-nil links